openedx/features/smart_referral/views.py

```diff
@@ -0,0 +1,23 @@
+from edx_rest_framework_extensions.auth.session.authentication import SessionAuthenticationAllowInactiveUser
+from rest_framework.decorators import api_view, authentication_classes, permission_classes
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework.status import HTTP_200_OK, HTTP_400_BAD_REQUEST
+
+from .serializers import SmartReferralSerializer
+from .tasks import task_referral_and_toolkit_emails
+
+
+@api_view(['POST'])
+@permission_classes([IsAuthenticated])
+@authentication_classes([SessionAuthenticationAllowInactiveUser, ])
+def send_initial_emails(request):
```

Comment:
This is telling us just one thing that we are sending email, but actually we are doing two things, we are also saving records in smart_refferal table let's please rename it to serve the purpose of the method.